examples/media_upload.py

```diff
@@ -17,7 +17,7 @@
 # https://developer.twitter.com/en/docs/ads/creatives/guides/media-library
 resource = '/1.1/media/upload.json'
 params = {
-    'additional_owners': '756201191646691328',
+    'additional_owners': ACCOUNT_ID,
```

Comment:
Great catch that this was hardcoded. This field represent the user ID of another user that should have access to use this asset. So this field is only necessary when the credentials that are uploading the asset are different from a user to an ads account that will use the asset. In this example, it isn't clear how to provide an `additional_owners` value that would make sense.

It might be best to comment out this parameter but add a note that this should be set for another user ID.